docs/appendix/troubleshooting.md

```diff
@@ -62,11 +62,24 @@ This is only required for the first `ssh` connection you make to a newly-imaged
 
 ### AppImages require FUSE to run
 
-**Description:** `viam-server` is distributed for Linux as an [AppImage](https://appimage.org/), which relies on FUSE (Filesystem-in-Userspace).
+**Related Error:** `dlopen(): error loading libfuse.so.2`
+
+**Description:** `viam-server` is distributed for Linux as an [AppImage](https://appimage.org/), which requires FUSE (Filesystem-in-Userspace) version 2.
 FUSE is included in almost all modern Linux distributions by default, but some older Linux distros or minimal installs might not provide it out of the box.
+For example, the latest Raspberry Pi OS (Debian GNU/Linux 12 bookworm) does not include FUSE with its default installation.
```

Comment:
Is this confirmed? Or is it just FUSE 2 not installed (and it actually does have FUSE 3?)

Reply:
Does have FUSE 3, I'll fix this language, thank you!